pynq_composable/composable.py

```diff
@@ -81,6 +81,32 @@ def _get_ip_name_by_vlnv(description: dict, vlnv: str) -> str:
             return k
     return None
 
+def _streamline_pipeline(pipe: list) -> list:
```

Comment:
please add the pytest for this function in a new file, do not reuse the test_composable.py